From 6b27144de1c2b5982b6f219ea2a364db845a00ea Mon Sep 17 00:00:00 2001
From: Mark Prins <1165786+mprins@users.noreply.github.com>
Date: Mon, 4 Jul 2022 18:26:59 +0200
Subject: [PATCH] Exclude some files from the build

---
 tasks/generate-info.js | 22 +++++++++++++++++++---
 1 file changed, 19 insertions(+), 3 deletions(-)

diff --git a/tasks/generate-info.js b/tasks/generate-info.js
index bcf89bedc..19c165782 100644
--- a/tasks/generate-info.js
+++ b/tasks/generate-info.js
@@ -69,9 +69,25 @@ function getPaths() {
     const walker = walk(sourceDir);
     walker.on('file', (root, stats, next) => {
       const sourcePath = path.join(root, stats.name);
-      if (/\.js$/.test(sourcePath)) {
-        paths.push(sourcePath);
-      }
+      if (
+          sourcePath.toLowerCase().includes('webgl') ||
+          sourcePath.toLowerCase().includes('geotiff') ||
+          sourcePath.toLowerCase().includes('gml') ||
+          sourcePath.toLowerCase().includes('format/filter') ||
+          sourcePath.toLowerCase().includes('format/xml') ||
+          sourcePath.toLowerCase().includes('capabilities') ||
+          sourcePath.toLowerCase().includes('format/wk') ||
+          sourcePath.toLowerCase().includes('iiif') ||
+          sourcePath.toLowerCase().includes('vectortile') ||
+          sourcePath.toLowerCase().includes('mapguide') ||
+          sourcePath.toLowerCase().includes('cartodb') ||
+          sourcePath.toLowerCase().includes('zommify') ||
+          sourcePath.toLowerCase().includes('mapbox')
+        ) {
+          // skip
+        } else {
+          paths.push(sourcePath);
+        }
       next();
     });
     walker.on('errors', () => {
-- 
2.34.1

