diff --git a/tasks/generate-info.js b/tasks/generate-info.js
index 281be0c1c0..5e028dbc5e 100644
--- a/tasks/generate-info.js
+++ b/tasks/generate-info.js
@@ -69,7 +69,29 @@ function getPaths() {
     const walker = walk(sourceDir);
     walker.on('file', (root, stats, next) => {
       if (stats.name.endsWith('.js')) {
-        paths.push(path.join(root, stats.name));
+        if (path.join(root, stats.name).toLowerCase().includes('webgl') ||
+            path.join(root, stats.name).toLowerCase().includes('geotiff') ||
+            path.join(root, stats.name).toLowerCase().includes('gml') ||
+            path.join(root, stats.name).toLowerCase().includes('format/filter') ||
+            path.join(root, stats.name).toLowerCase().includes('format/xml') ||
+            path.join(root, stats.name).toLowerCase().includes('wmscapabilities') ||
+            path.join(root, stats.name).toLowerCase().includes('format/esrijson') ||
+            path.join(root, stats.name).toLowerCase().includes('format/wk') ||
+            path.join(root, stats.name).toLowerCase().includes('iiif') ||
+            path.join(root, stats.name).toLowerCase().includes('vectortile') ||
+            path.join(root, stats.name).toLowerCase().includes('mapguide') ||
+            path.join(root, stats.name).toLowerCase().includes('cartodb') ||
+            path.join(root, stats.name).toLowerCase().includes('zoomify') ||
+            path.join(root, stats.name).toLowerCase().includes('mapbox') ||
+            path.join(root, stats.name).toLowerCase().includes('sentinelhub') ||
+            path.join(root, stats.name).toLowerCase().includes('flow') ||
+            path.join(root, stats.name).toLowerCase().includes('geozarr') ||
+            path.join(root, stats.name).toLowerCase().includes('ogcmap')) {
+          // skip adding these files to the paths array
+          console.log(`Skipping ${path.join(root, stats.name)}`);
+        } else{
+          paths.push(path.join(root, stats.name));
+        }
       }
       next();
     });
